activities/TankOp.activity/css/activity.css

```diff
@@ -18,4 +22,132 @@
   position: absolute;
   visibility: hidden;
   z-index: 5;
-} 
\ No newline at end of file
+}
+
+.tutorial-title {
+	background-color: #808080 !important;
```

Comment:
Do not mix indentation style in the same file

Reply:
Indention Issue Fixed in activity.css file